src/pq_flash_index.cpp

```diff
@@ -1178,7 +1178,6 @@ void PQFlashIndex<T, LabelT>::cached_beam_search(const T *query1, const _u64 k_s
     unsigned cmps = 0;
     unsigned hops = 0;
     unsigned num_ios = 0;
-    unsigned k = 0;
```

Comment:
where is this extra variable used?

Reply:
no code uses this variable. our internal building pipeline catches this